INSTALL_DOCKER.rst

```diff
@@ -35,15 +32,15 @@ If your user does not already have ``sudo`` rights, you should add your user to
 
   ::
 
-    usermod -G wheel -a <USERNAME>
+    $ usermod -G wheel -a <USERNAME>
 
 This change will take effect once you have logged out as root and logged back in as <USERNAME>.
```

Comment:
Actually @jodh-intel , as you mention it ;-), I was going to add a note or Issue saying that I'm not sure the 'newgrp' is the right way to do this (below) - each newgrp, at least on my F24 box, starts a new sub-shell, so at the end of the whole sequence I need to 'exit' 3 times to get out of my original shell - not great. I was going to ask @gorozco1 if he had any other ideas here - apart from log-out and in again. I don't remember if there is a nice way to re-initialise a login shell etc. Worst case is I may add a note explaining what the `newgrp` is doing.
Oh - and as to the above - I wondered about adding `newgrp` below the above - *but* - the above is done at a root login, and then it asks you to log out and then back in as USERNAME - so you don't need a `newgrp` here I think.